### DISCUSSION TITLE
Disable PHP 8 check possible for Kimai V2?

### DISCUSSION THREAD

**Discussion by samuelT2** [kimai/kimai#4039] [Questions & Answers] [answered]
Hi

Default PHP version set by my hoster is still 7.4.33. I can overwrite this by adding ```AddHandler application/x-httpd-php82 .php``` to a .htaccess file. I did that for Kimai 1.x and it worked fine. During the upgrade process for Kimai 2, this happened:

![Screenshot 2023-05-20 165446](https://github.com/kimai/kimai/assets/10983143/e93e1410-7e6d-4715-a069-6646c6b35001)

Can I somehow disable this check?

**Comment by samuelT2** [comment]
Re-installed v1.30.11. Doctor says, PHP 8.2.3 is running - would be nice if I can disable this check somehow.

![image](https://github.com/kimai/kimai/assets/10983143/d41a602c-0cc7-43ff-995d-55c4f2769b22)

**Reply by kevinpapst** [reply, marked as answer]
That doesn't make any sense. Kimai needs PHP > 8.1 - running it with a lower version will lead to bugs/errors whatever.
You have to use the correct php version. 

Your hoster has likely a way to use another PHP version in SSH (e.g. do not use `php` but `php81`). 
Read the installation docs and find some ideas how to address your problem. There are chapters about different hosters, which cover many scenarios.

**Reply by samuelT2** [reply]
Jep, [it does](https://www.cyon.ch/support/a/php-standardversion-fur-die-kommandozeile-festlegen) - thank you for the hint. Upgrade to v2 worked flawless (only the MySql user needed a privilege change).

**Comment by samuelT2** [comment]
I can't find it in your docs, is there somewhere described which rights my database user needs? That's what's currently granted to my user:

![image](https://github.com/kimai/kimai/assets/10983143/8cb2ab00-cf35-49d8-a9e9-7b5e6e518da4)
